### PR TITLE
feat(icons): `FeatherIcons` add full icon pack

### PR DESCRIPTION
**Pull request by waterplea**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by splincode** [comment]
image

`$ nx test kit`

**Review by splincode — APPROVED** [review]

**Comment by splincode** [comment]
image

**Comment by splincode** [comment]
@waterplea what do you think?
image

File: projects/demo/src/modules/app/app.template.html [hunk 1]
```diff
@@ -12,22 +12,40 @@
...
+                tuiLink
+                href="https://discord.gg/zrB2EdJjEy"
+                title="Get help on Angular Discord"
+                iconAlign="left"
+                rel="noreferrer"
+                icon="assets/icons/discord.svg"
+                target="_blank"
+                class="link"
+            >
+                Discord
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/app.template.html:34]: Let's make them to use the similar colors?
blue

Comment (nsbarsukov) [reply projects/demo/src/modules/app/app.template.html:34]: for example
black

File: projects/demo/src/modules/customization/icon-set/examples/1/index.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="b-form">
     <tui-notification>
         As usual with the DI this is hierarchical. Meaning you can provide different
-        <code>TUI_ICONS_PATH</code>
```
Comment (nsbarsukov) [inline projects/demo/src/modules/customization/icon-set/examples/1/index.html:4]: Let's change it in Stackblitz Starter too

https://github.com/Tinkoff/taiga-ui/blob/a0093a010a9263f2880c5e10fafd3df7c76c657a/projects/demo/src/modules/app/stackblitz/project-files/src/app/app.module.ts.md?plain=1#L18-L22

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**